Webapp/sources/blueprints/reaction_table/routes.py

```diff
@@ -24,17 +24,16 @@
 
 from . import \
     reaction_table_bp  # imports the blueprint of the reaction table route
-
-if not current_app.testing:
-    try:
-        from STOUT import translate_forward
-    except:
-        pass
+#
```

Comment:
Should this be commented out, or deleted if the STOUT approach is no longer used?
The `iupac_convert` function that uses it needs refactoring and documenting anyway (along with this testing approach), but if this is a temporary fix, please do a comment in the code describing why.

Reply:
Thanks for spotting, this change shouldn't have been pushed. I commented it out just to speed up loading the app start up whilst developing.